lib/addons/karpenter/index.ts

```diff
@@ -0,0 +1,131 @@
+import { Construct } from '@aws-cdk/core';
+import { Role, ManagedPolicy, ServicePrincipal, CfnInstanceProfile, PolicyDocument } from '@aws-cdk/aws-iam';
+import { ClusterInfo } from '../../spi';
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from '../helm-addon';
+import { createNamespace, setPath, createServiceAccount, conflictsWith, tagSubnets } from '../../utils'
+import { KarpenterControllerPolicy } from './iam'
+
+/**
+ * Configuration options for the add-on
+ */
+interface KarpenterAddOnProps extends HelmAddOnUserProps {
+    /**
+     * Specs for Default Provisional (Optional)
+     */
+    defaultProvisionerSpecs?: { 
```

Comment:
I think this should be called `provisionerSpecs` in the interface.

Reply:
revised.